### PR TITLE
RDM-11095: Revert message publisher flux changes (aat, prod)

### PR DESCRIPTION
**Pull request by kiran-yenigala-hmcts**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RDM-11097

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ X] Yes
[ ] No
```

### CODE REVIEW

**Comment by hemantt** [comment]
@kiran-yenigala-hmcts can we add the changes to other environments as well please?

File: k8s/prod/common/ccd/data-store-api.yaml [hunk 1]
```diff
@@ -19,18 +19,6 @@ spec:
   values:
     java:
       replicas: 9
-      keyVaults:
-        ccd:
-          secrets:
-          - data-store-api-POSTGRES-USER-V11
```
Comment (smathangi) [inline k8s/prod/common/ccd/data-store-api.yaml:25]: remove DATA_STORE_DB_USERNAME: ccd@ccd-data-store-api-postgres-db-v11-prod entry as bootstrap has that already.

**Comment by smathangi** [comment]
Please clean-up aat flux as well.

**Review by smathangi — APPROVED** [review]
LGTM

**Pull request merged**